### ISSUE TITLE
Could you make the getting started more newcomer friendly?

### ISSUE THREAD

**Issue by ditoslav** [strongloop-community/loopback-connector-sqlite3#36]
So I have just installed loopback and wanted to connect it to sqlite. I googled "sqlite loopback" and found the getting started. I do not really understand how to do the steps. 
Where do I put the things mentioned in "Connector settings" ?
Where do I put the text specified in the "Model definition" section?

Thanks

**Comment by ditoslav** [comment]
Also, why isnt sqlite listed in the `lb datasource` list of databases?

**Comment by jesusgp22** [comment]
I will +1 this, so far it's being impossible to get it to work

**Comment by juanvillegas** [comment]
I agree. However, remember LB3 is mostly EOL so I wouldn't expect them to improve any of this, not even through pull requests.
